tests/Traits/TPresenterTest.phpt

```diff
@@ -259,6 +259,13 @@ class TPresenterTest extends \Testbench\CustomPresenterTestCase
 		}, 'Tester\AssertException', "field 'test' should be defined as required, but it's not");
 	}
 
+    public function testUserLoggedIn()
+    {
```

Comment:
Please check that user is not logged in at the beginning of the test. Test scenario should be:
- not logged in (test)
- login
- logged in (test)
- logout
- not logged in (test)